Source/NSDocumentController.m

```diff
@@ -955,7 +956,7 @@ - (IBAction) saveAllDocuments: (id)sender
 
 - (IBAction) openDocument: (id)sender
 {
-  NSError *err = nil;
+  //NSError *err = nil;
```

Comment:
Not sure that this line should be commented out.

Reply:
Moved to #else section and uncommented